### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by kuludawar**
I have created Checkout.module.ts file.

### CODE REVIEW

**Review by kuludawar — COMMENTED** [review]
I did some change in Checkout.module.ts file.

File: src/app/spartacus/spartacus-configuration.module.ts [hunk 1]
```diff
@@ -18,12 +18,29 @@ import { defaultCmsContentProviders, layoutConfig, mediaConfig } from "@spartacu
...
     },
-  }), provideConfig(<I18nConfig>{
+    //<I18nConfig>
+  }), provideConfig({
     i18n: {
       resources: translations,
-      chunks: translationChunksConfig,
-      fallbackLang: 'en'
-    },
+        chunks: translationChunksConfig,
```
Comment (timofeysie) [inline src/app/spartacus/spartacus-configuration.module.ts:25]: It would be better if you made two separate commits, one for the guest checkout, and another for the i18n changes.  In this way, if there is a problem with either, they can be dealt with separately.

Comment (timaccen) [inline src/app/spartacus/spartacus-configuration.module.ts:25]: It would be better if you made two separate commits, one for the guest checkout, and another for the i18n changes. In this way, if there is a problem with either, they can be dealt with separately.
